rosplan_planning_system/launch/includes/dispatch_interface.launch

```diff
@@ -9,6 +9,8 @@
 	<arg name="action_feedback_topic"    default="action_feedback" />
 	<!-- visualize edges with colors: interference edge, conditional edge, etc -->
 	<arg name="display_edge_type"        default="false" />
+    <arg name="timeout_actions"          default="true" />
```

Comment:
Set this to false as default to prevent existing demos/projects from failing due to strict timeout on PDDL action duration.